CHANGELOG.md

```diff
@@ -11,6 +11,106 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.7.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.7.0. This release brings new features such as:
+- Complete GPU-resident sparse direct solvers feature set and interfaces,
+- Sparse batched iterative kernels support for Dense and ELL matrix types and the BiCGSTAB solver,
+- A new MC64 reordering,
+- MPI support for the SYCL backend,
+- Improved Cholesky factorization performance,
+- Improved ParILU(T)/ParIC(T) preconditioner convergence,
+and more!
+
+If you face an issue, please first check our [known issues page](https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues) and the [open issues list](https://github.com/ginkgo-project/ginkgo/issues) and if you do not find a solution, feel free to [open a new issue](https://github.com/ginkgo-project/ginkgo/issues/new/choose) or ask a question using the [github discussions](https://github.com/ginkgo-project/ginkgo/discussions).
+
+Supported systems and requirements:
++ For all platforms, CMake 3.16+
++ C++14 compliant compiler
++ Linux and macOS
+  + GCC: 5.5+
+  + clang: 3.9+
+  + Intel compiler: 2019+
+  + Apple Clang: 14.0 is tested. Earlier versions might also work.
+  + NVHPC: 22.7+
+  + Cray Compiler: 14.0.1+
+  + CUDA module: CMake 3.18+, and CUDA 10.0+ or NVHPC 22.7+
+  + HIP module: ROCm 4.5+
+  + DPC++ module: Intel OneAPI 2021.3+ with oneMKL and oneDPL. Set the CXX compiler to `dpcpp` or `icpx`.
```

Comment:
Does 2021.3 still work ? I think we only test from 2022.1 onwards ?

Reply:
I'll update it, indeed we don't know for sure